src/main/java/dto/quotes.java

```diff
@@ -13,6 +13,6 @@ class quotes {
     private double bidPrice;
     private int bidSize;
     private String timestamp;
-    private String condition[];
+    private String[] condition;
```

Comment:
Changed Array declarations from C-style to be Java style declarations

Reply:
Covered in a previous code review.  When that occurs, first in wins.